frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -18,14 +17,24 @@ const Container = styled.div`
   border-width: 2px;
   border-radius: 7px;
   background-color: var(--light-gray);
+  margin:5rem 0 ;
+  margin-bottom:4;
 `;
 
+const TwoParamContainer = styled.div`
+display: flex;
+flex-direction: column;
+height:100%;
+align-items:center;
+padding-top:2rem;
+`
```

Comment:
I would suggest installing prettier to fix some of these format issues in javascript files

Reply:
I had it installed but I had to also set it as default for javascript files